django/template/defaulttags.py

```diff
@@ -578,10 +615,14 @@ def __repr__(self):
         return "<WithNode>"
 
     def render(self, context):
+        return ''.join(self.stream(context))
```

Comment:
Would it make more sense for the default behavior of ``Node.render()`` to be ``return ''.join(self.stream(context))``?. It seems that tags should be able to get away with just defining a ``stream`` method.